src/PKSim.Infrastructure/ProjectConverter/v7_3/Converter721To730.cs

```diff
@@ -100,6 +121,34 @@ private void convertProject(PKSimProject project)
          _converted = true;
       }
 
+      private void convertFormulation(Formulation formulation)
+      {
+         if (formulation == null)
+            return;
+
+         var templateFormulation = _formulationRepository.FormulationBy(formulation.FormulationType);
+         updateIsInputStateByNameAndValue(formulation, templateFormulation);
+      }
+
+      private void convertEvent(PKSimEvent pkSimEvent)
+      {
+         if (pkSimEvent == null)
+            return;
+
+         var templateEvent = _eventGroupRepository.FindByName(pkSimEvent.TemplateName);
+         updateIsInputStateByNameAndValue(pkSimEvent, templateEvent);
+      }
+
+      private void updateIsInputStateByNameAndValue(IContainer containerToUpdate, IContainer templateContainer)
+      {
+         foreach (var templateParameter in templateContainer.AllParameters(x => x.Visible && x.Editable && x.ValueIsDefined()))
+         {
+            var parameter = containerToUpdate.Parameter(templateParameter.Name);
+            if (parameter != null && !ValueComparer.AreValuesEqual(parameter, templateParameter))
```

Comment:
and value has changed! In that case this should be marked as IsDefault=false

Reply:
Parameter.IsDefault=false should be also set  in case TemplateParameter.IsDefault=false

Reply:
It's already done before for all parameters in the container. This is just the specifics for types

Reply:
OK,  I see